cpp/include/cudf/detail/utilities/hash_functions.cuh

```diff
@@ -593,3 +595,6 @@ struct IdentityHash {
 
 template <typename Key>
 using default_hash = MurmurHash3_32<Key>;
```

Comment:
Should the `default_hash` be in a `detail` namespace or not? I put the hash function implementations like `MurmurHash3_32` in a `detail` namespace.

Reply:
My recollection is that the `default_hash` is only used as the default hash function for various internal hash tables (join, groupby, etc), so it's only necessary for developers. As such, `detail` seems appropriate.